python/build-wheel-macos.sh

```diff
@@ -79,6 +79,8 @@ for ((i=0; i<${#PY_VERSIONS[@]}; ++i)); do
     $PIP_CMD install -q numpy==$NUMPY_VERSION cython==0.29.0
     # Install wheel to avoid the error "invalid command 'bdist_wheel'".
     $PIP_CMD install -q wheel
+    # Set the commit SHA in __init__.py.
+    sed -i.bak "s/{{RAY_COMMIT_SHA}}/$TRAVIS_COMMIT/g" ray/__init__.py && rm ray/__init__.py.bak
```

Comment:
I'm almost sure that I will forget to modify this when migrating to github actions. So maybe we can do:

```bash
if [ -n "$TRAVIS_COMMIT" ]; then # if TRAVIS_COMMIT is set
    sed ...
else
   echo "TRAVIS_COMMIT is not set, it's needed to python __commit__ field"
   exit 1
fi
```

Reply:
done